webclient/webclient/src/main/java/io/helidon/webclient/RequestConfiguration.java

```diff
@@ -50,13 +52,18 @@ WebClientServiceRequest clientServiceRequest() {
         return services;
     }
 
+    public long requestId() {
```

Comment:
public method in a non-public class, also no javadoc

Reply:
ehm... yes.. This should be package private